plop-templates/request.hook.hbs

```diff
@@ -2,19 +2,37 @@
 import { useState, useEffect } from 'react';
 {{else}}import React, { useState, useEffect } from 'react';
 {{/isTypescript}}
-export function fetchData(url, setValue) {
+export function fetchData(url, setValue, setLoading, setError) {
   return fetch(url)
     .then((response) => response.json())
-    .then((json) => setValue(json));
+    .then((json) => {
+      setLoading(true);
+      setValue(json);
+    })
+    .catch((error) => {
+      setLoading(true);
+      setError(error.message);
+    });
```

Comment:
This logic is wrong.  It should be setIsLoading then once it has finished turn that flag off.

Reply:
Fixed.